MvvmCross/ViewModels/MvxAppStart.cs

```diff
@@ -91,24 +99,42 @@ protected virtual void NavigateToFirstViewModel(object hint)
         }
     }
 
-    public class MvxAppStart<TViewModel, TParameter> : MvxAppStart<TViewModel> where TViewModel : IMvxViewModel<TParameter>
+    public class MvxAppStart<TViewModel, TParameter> : MvxAppStart<TViewModel> where TViewModel : IMvxViewModel
```

Comment:
Because developers may want to use the generic MvxApplication (so they get a strongly typed hint there) without having to use the generic IMvxViewModel

Reply:
Legit!